app/src/atoms/AtomInput.re

```diff
@@ -0,0 +1,18 @@
+module Styles = {
+  open Css;
+  open Particle;
+
+  let input =
+    style([
+      border(px(1), solid, Color.lightGray()),
+      display(block),
+      padding(px(8)),
+      borderRadius(px(4)),
+      width(pct(100.0)),
+      height(px(48)),
+    ]);
+};
+
+[@react.component]
+let make = (~placeholder="", ~onChange=_ => ()) =>
```

Comment:
Proper type the `onChange` similar to `onClick` would be great.

Reply:
Ditto. https://github.com/oscoin/mvp/pull/14#discussion_r318096033